pkg/machines/components/vmDisksTabLibvirt.jsx

```diff
@@ -25,8 +25,9 @@ import VmDisksTab from './vmDisksTab.jsx';
 import { DiskSourceCell, DiskExtras } from './vmDiskColumns.jsx';
 
 class VmDisksTabLibvirt extends React.Component {
-    componentWillMount() {
-        this.props.onUsageStartPolling();
+    constructor(props) {
+        super(props);
+        props.onUsageStartPolling();
```

Comment:
Wouldn't it make more sense to start the polling in componentDidMount()?

Reply:
It would, but it was before first render, so I though I would keep it that way. I guess there should be harm done, right @KKoukiou

Reply:
Ack, can happen as a separate PR then.